templates/opencraft-nginx.j2

```diff
@@ -30,6 +30,17 @@ server {
     access_log /var/log/nginx/{{ OPENCRAFT_DOMAIN_NAME }}_access.log;
     error_log /var/log/nginx/{{ OPENCRAFT_DOMAIN_NAME }}_error.log error;
 
+    # serve static files
+    #
+    location /static {
+        alias {{ OPENCRAFT_STATIC_FILES_DIR }};
+        try_files $uri $uri/ =404;
```

Comment:
@smarnach oh, I forgot to get rid of this.

I was using `autoindex on;` before.
Thanks for spotting that!